tutorials/System.getHostSpecs.md

```diff
@@ -0,0 +1,6 @@
+Retrieves system information.
+
+# Example
+```js
+ fin.System.getHostSpecs().then(specs => console.log(specs)).catch(err => console.log(err));
```

Comment:
Can be shorter: `fin.System.getHostSpecs().then(console.log).catch(console.error);`